app/controllers/pushes_controller.rb

```diff
@@ -52,13 +55,15 @@ def validate_ids
     if requested_ids.count > 1
       requested_ids.each_with_index do |id, index|
         next if index == (requested_ids.count-1)
-        query += %(#{id} OR )
+        query += %(id:#{id} OR )
       end
     end
 
-    query += %(#{requested_ids.last})
-    query_params = {q: query}
-    response, response_documents = search_results(query_params)
+    query += %(id:#{requested_ids.last})
```

Comment:
Same as above...  I think lines 55-62 can be reduced down to...
```
query += requested_ids.map { |id| "id:#{id}" }.join(" OR ")
```

Reply:
👍